guides/common/modules/proc_configuring-ms-dhcp.adoc

```diff
@@ -0,0 +1,34 @@
+[id="Configuring_MS_DHCP_{context}"]
```

Comment:
As noted before: do we want to document Windows? Right now I'm leaning to moving it over to smart-proxy.git.

Reply:
Moved for now; but as said before, I understand if we decide not to add this to the official Foreman documentation! (adding this is most certainly not my own idea...)